### PR TITLE
Update Transaction list to display USDS transfers

### PR DESCRIPTION
**Pull request by duy-nguyentuong**

### CODE REVIEW

File: ts/prime/Web/src/components/transactionList.tsx [hunk 1]
```diff
@@ -105,6 +124,7 @@ const TransactionList = ({ initLoading, loading, list, loadMore, pageTitle }: Tr
       dataSource={list}
       loadMore={loadMore}
       renderItem={(item) => (
+        // TODO: link to etherscan for USDS transfers
```
Comment (Git-on-my-level) [inline ts/prime/Web/src/components/transactionList.tsx:127]: Here's the format for a Etherscan URL https://etherscan.io/tx/0x50d9794c55ee0ad3d38a34c943d03c5b6df4e62f8ffe8696774f79c56c758c1b
The last part is the tx hash which you will fill in
For prod use the main etherscan page, for everything else use ropsten.etherscan.io

**Comment by duy-nguyentuong** [comment]
Screen Shot 2020-07-29 at 2 37 46 PM

File: ts/prime/Web/src/components/transactionList.tsx [hunk 2]
```diff
@@ -11,47 +11,67 @@ const SIGN_MINUS = '-'
...
-    direction = 'From'
-    if (transaction.method === 'wire') {
-      return 'Deposit'
-    }
-    // TODO: Handle other deposit methods (Eg: USDS)
-  }
+  switch (transaction.method) {
+    case TRANSACTION_METHOD_USDS:
+      direction = transaction.type === TRANSACTION_TYPE_WITHDRAW ? 'To' : 'From'
+      const usdsTransactionInfo = direction == 'To' ? transaction.to : transaction.from
```
Comment (markusait) [inline ts/prime/Web/src/components/transactionList.tsx:39]:
```javascript
// Can be simmplified
const usdsTransactionInfo =  transaction.type === TRANSACTION_TYPE_WITHDRAW ? transaction.to : transaction.from
```

**Comment by duy-nguyentuong** [comment]
> When I do a USDS withdrawal, a Stably Prime transfer is shown. Instead, it should state that this is a pending USDS withdrawal and then update the record once it is settled.

Did you checkout `go-packages` to `update_transaction_history`, @markusait?

**Comment by markusait** [comment]
@duy-nguyentuonga I thought this was already merged stablyio/go-packages#337   
But I checked your branch out and the behavior still persists. Generally, you would need to  
check the Token transfer tables first before the account cash transfer tables.   
On a separate note, if you think it's useful to have USDS withdrawal working in beta without having to clearing the account cash transfer let me know.

File: ts/prime/Web/src/components/transactionList.tsx [hunk 3]
```diff
@@ -11,47 +11,67 @@ const SIGN_MINUS = '-'
...
   wire: 'Wire',
   ach: 'ACH',
   prime: 'Stably Prime',
+  usds: 'USDS',
 }
 const TRANSACTION_STATUS_COLOR_MAP = {
   settled: '#35b8a5',
   pending: '#2889ca',
   cancelled: '#fe0a00',
+  link: '#0B0080',
```
Comment (chauhannishith) [inline ts/prime/Web/src/components/transactionList.tsx:29]: try to put all colors in one file and import from there

**Comment by duy-nguyentuong** [comment]
I'm gonna merge this PR for QA, any change request or feedback should be fixed in the further follow PR.

**Pull request merged**